### PR TITLE
add german translation

### PR DESCRIPTION
**Pull request by Stzyxh**
yes

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be `Der` instead of `der` (beginning of sentence)

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be `Ihre` instead of `ihre` (beginning of sentence)

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be `Kein` instead of `kein`, and there should be a `.` at the end of the sentence (which is present in the original string as well)

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be `Erneut` instead of `erneut`

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: There shouldn't be an empty space at the end

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be `Abbrechen` instead of `abbrechen`

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be `durchsuchen` instead of `dursuchen`

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be `Keine` instead of `keine`, and there should be a `.` at the end

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Do we need the `von` here? I've never heard of this unit in german but according to google, it's `Zoll Quecksilber`

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be `Hell` instead of `hell`

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be `Kilometer` instead of `Killometer`

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be capitalized

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be capitalized

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be `Konstant`

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be `Tatsächlich %s`

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: This should indicate that it describes the felt temperature instead of the actual, like in the English strings.
Example: `Durch den Wind fühlt es sich kühler an.`

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be `UV-Index`

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be `Verwenden Sie Sonnenschutz von %1$s bis %2$s.`

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should (probably) be `Verwenden Sie Sonnenschutz ab %s.`, depending on the context

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be `Verwenden Sie Sonnenschutz bis %s.`

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: `Stark eingeschränkte Sicht.` would probably sound better here

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be translated: `Böen um %s.`

**Review by SehrGuterCode — CHANGES_REQUESTED** [review]
There are a few things that should be fixed here (Source: I'm a German native speaker)

File: app/src/main/res/values-de/strings.xml
Comment (Stzyxh) [reply app/src/main/res/values-de/strings.xml]: Thx i fixed all that things

**Comment by davidtakac** [comment]
Thank you for collaborating on this. Since I'm not a native speaker, I'll wait for @SehrGuterCode's approval before merging. I can also fix the merge conflicts due to recently merged Chinese (Simplified).

File: app/src/main/java/com/davidtakac/bura/common/Locale.kt
Comment (SehrGuterCode) [inline app/src/main/java/com/davidtakac/bura/common/Locale.kt]: Is this supposed to be lowercased?

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: Should be `Extrem`

File: app/src/main/res/values-de/strings.xml
Comment (SehrGuterCode) [inline app/src/main/res/values-de/strings.xml]: There should be a `.` at the end of the sentence here

**Review by SehrGuterCode — CHANGES_REQUESTED** [review]

**Review by SehrGuterCode — APPROVED** [review]
LGTM!

**Pull request merged**
